doc/design/cluster_train/PServer_design_doc.md

```diff
@@ -0,0 +1,261 @@
+# Design Doc: Parameter Server Process
```

Comment:
We are not designing a process, but designing a server program. Therefore the title should be

> Design Doc: Parameter Server